setup.py

```diff
@@ -16,13 +16,13 @@
 from version import VERSION  # NOQA
 
 INSTALL_REQUIRES = [
-    'pandas >= 0.14',
-    'numpy >= 1.8',
+    'pandas >= 0.14, < 0.25',
```

Comment:
Im not sure these caps are required TBH. Based on my testing, if the user has a non outdated `pip` then pip will be smart enough to only take the correct version of each dependency for the users current python version anyways